Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -345,7 +345,9 @@ def identitySim(wf):
     'CMSSW_7_5_0_pre6-PU25ns_75X_mcRun2_asymptotic_v1-v1',  # 5 - fullSim PU 25ns premix
     'CMSSW_7_5_0_pre6-PU50ns_75X_mcRun2_startup_v1-v1',     # 6 - fullSim PU 50ns premix
     'CMSSW_7_5_0_pre6-75X_mcRun2_asymptotic_v1_FastSim-v1', # 7 - fastSim premix
-    'CMSSW_7_5_0_pre6-75X_mcRun2_HeavyIon_v1-v1'            # 8 Run2 HI GEN-SIM
+    'CMSSW_7_5_0_pre6-75X_mcRun2_HeavyIon_v1-v1',           # 8 Run2 HI GEN-SIM
+    'CMSSW_7_5_0_pre5-MCRUN2_75_V5_FastSim-v1',             # 9 - fastSim mb
```

Comment:
@srimanob Hi Phat, this parameter is actually the same as parameter " 7 - fastSim premix" , it is repeated. Could you please either remove this one or remove the " 7 - fastSim premix" and replace the things with this "9 - fastSim mb" ? 
Thanks, Hengne